dev/openchpl_preload.sql

```diff
@@ -1977,8 +1977,9 @@ INSERT INTO user_permission (user_permission_id, "name", description, authority,
 (2, 'ACB' ,'This permission gives a user write access to their ACBs.', 'ROLE_ACB' , -1),
 (4, 'ATL' ,'This permission gives a user write access to their ATLs.', 'ROLE_ATL' , -1),
 (6, 'CMS_STAFF' ,'This permission gives a user read access to CMS reports.', 'ROLE_CMS_STAFF' , -1),
-(7, 'ONC', 'This permission gives ONC users administrative privileges.', 'ROLE_ONC', -1);
-SELECT pg_catalog.setval('user_permission_user_permission_id_seq', 8, true);
+(7, 'ONC', 'This permission gives ONC users administrative privileges.', 'ROLE_ONC', -1),
+(8, 'DEVELOPER', 'This permission gives Developer Users administrative privileges.', 'ROLE_DEVELOPER', -1);
```

Comment:
I know we don't use the description field but I was thinking it should be written more like the ACB/ATL permission... "This permission gives a user write access to their Developers".